contrib/terraform/openstack/modules/compute/main.tf

```diff
@@ -90,8 +91,8 @@ resource "openstack_compute_instance_v2" "bastion" {
 }
 
 resource "openstack_compute_instance_v2" "k8s_master" {
-  name              = "${var.cluster_name}-k8s-master-${count.index+1}"
   count             = "${var.number_of_k8s_masters}"
+  name              = "${var.cluster_name}-k8s-master-${count.index+1}"
```

Comment:
Is it necessary to change the order or `count` and `name` ? (this appears in a couple of other places as well)

Reply:
Hi @Miouge1. Thanks for the review. I cleaned up the commit to address your comments.